### PR TITLE
v5 release

### PR DESCRIPTION
**Pull request by mikedanese**
@jdef

### CODE REVIEW

File: data/datafile.go [hunk 1]
```diff
@@ -450,7 +450,7 @@ func appAssetsJsAppJs() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "app/assets/js/app.js", size: 43468, mode: os.FileMode(420), modTime: time.Unix(1450202755, 0)}
+	info := bindataFileInfo{name: "app/assets/js/app.js", size: 43468, mode: os.FileMode(416), modTime: time.Unix(1454014512, 0)}
```
Comment (jdef) [inline data/datafile.go:453]: any reason that these file perms are expressed as decimal instead of something more easily understood (e.g. octal `0640`)?

**Comment by jdef** [comment]
xref https://github.com/kubernetes/kubernetes/issues/20285

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@
 .PHONY: all bindata container push clean
 
 # Keep this at dev, so no one accidentally blows away the latest published version.
-TAG = v4 # current version: v4
+TAG = v5 # current version: v4
```
Comment (jdef) [inline Makefile]: s/v4/v5/

**Comment by jdef** [comment]
minor comments, otherwise lgtm

File: data/datafile.go (hunk 1 above)
Comment (mikedanese) [inline data/datafile.go:453]: This file is autogenerated so not sure. Also not sure why the perms changed...

Comment (mikedanese) [inline data/datafile.go:453]: Actually I would guess that somebody changed the permission on that file in another commit

**Comment by jdef** [comment]
@mikedanese Isn't there supposed to be a manual `npm run build` step that would have generated additional changes for the data file here?

**Pull request merged**